contrib/registry/nacos/registry.go

```diff
@@ -82,7 +83,7 @@ func New(cli naming_client.INamingClient, opts ...Option) (r *Registry) {
 // Register the registration.
 func (r *Registry) Register(_ context.Context, si *registry.ServiceInstance) error {
 	if si.Name == "" {
-		return fmt.Errorf("kratos/nacos: serviceInstance.name can not be empty")
+		return errors.New("kratos/nacos: serviceInstance.name can not be empty")
```

Comment:
Would it be better to declare as a variable?

Reply:
What does it mean to 'declare as a variable'? means `fmt.error` better?

Reply:
a global variable such as "var Errxxxxxx = errors.New("xxxxx")".

Reply:
> a global variable such as "var Errxxxxxx = errors.New("xxxxx")".

modified @Casper-Mars